src/libertem/udf/masks.py

```diff
@@ -38,49 +81,83 @@ class ApplyMasksUDF(UDF):
 
     .. versionadded:: 0.4.0
     '''
+    # '''
```

Comment:
Is this commented docstring needed for some reason? If so, why?

Reply:
No, this was a leftover from debugging a docstring issue. Thx! :-)